1.0-draft/index.html

```diff
@@ -488,8 +495,8 @@ <h3>Authentication</h3>
     <section>
       <h2>Reconciliation Queries</h2>
```

Comment:
```suggestion
      <h2>Match Service</h2>
```

Reply:
Yes. This fits nicely with the other sections making them more consistent. Great suggestion. Thx.